crud.py

```diff
@@ -1,18 +1,149 @@
-from db import async_session, engine
+"""
+crud.py
```

Comment:
maybe we need 2 additional endpoints
- delete a task
- finish task (basically shorthand for update(is_finished=true))